java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/appetizer/AppetizerRepository.java

```diff
@@ -89,4 +89,15 @@ public List<Appetizer> findAllByIds(List<Integer> appetizerIds) {
 
         return namedParameterJdbcTemplate.query(sql, parameters, new BeanPropertyRowMapper<>(Appetizer.class));
     }
+
+    public List<Appetizer> findAppetizerThroughAppetizerOrdered(Integer id) {
+
+        String sql = "SELECT appetizer.* " +
+                " FROM appetizer" +
+                " JOIN appetizer_ordered ON appetizer.id = appetizer_ordered.appetizer_id" +
+                " JOIN \"order\" ON \"order\".id = appetizer_ordered.order_id" +
+                " WHERE \"order\".id = ?";
```

Comment:
same thing here, I think we don't need to join the order table

Reply:
this is been condensed, thanks